public/app/features/alerting/unified/components/rule-editor/labels/LabelsButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
I moved these two components to a separate file, as from now, we are using them from two different components.